IPython/core/completer.py

```diff
@@ -1517,7 +1517,13 @@ def _completions(self, full_text: str, offset: int)->Iterator[Completion]:
 
         for jm in jedi_matches:
             delta = len(jm.name_with_symbols) - len(jm.complete)
-            yield Completion(start=offset - delta, end=offset, text=jm.name_with_symbols, type=jm.type, _origin='jedi')
+            try:
+                type_ = jm.type
+            except Exception:
```

Comment:
Probably not,  in case there are other similar issues I don't want to crash IPython completion, and make the user experience horrible. I prefer to silently fail (unless --debug is set).

I can make it more specific while on master for users to actually report bugs though.